packages/common-frontend/src/components/DendronNote.tsx

```diff
@@ -36,5 +36,5 @@ export function DendronNote({ noteContent, config, ...rest }: Props) {
   useMermaid({ config, noteRenderedBody: noteContent });
   const logger = createLogger("DendronNote");
   logger.info({ ctx: "DendronNote", config });
-  return <div dangerouslySetInnerHTML={{ __html: noteContent }} {...rest} />;
+  return <><div dangerouslySetInnerHTML={{ __html: noteContent }} {...rest} /></>;
```

Comment:
do we need the `<></>` wrappers around the dvi? if so, can we add a comment explaining the function?